spec/unit/provider/package/zypper_spec.rb

```diff
@@ -209,14 +200,14 @@ def shell_out_expectation!(command, options=nil)
       shell_out_expectation!(
```

Comment:
I believe the title here has a typo - should be purge_package instead of remove_package